src/main/java/com/priko/calc/ShuntingClass.java

```diff
@@ -0,0 +1,159 @@
+package com.priko.calc;
+
+import java.util.*;
+import java.util.regex.Pattern;
+
+
+public class ShuntingClass {
+    private String eq;
+    private Stack<Integer> operatorStack = new Stack<>();
+    private StringBuilder output = new StringBuilder();
+    private Map<String, String> vars = new HashMap<>();
+
+    /**
+     *         Operators
+     *  [+] -   (Plus)   - [0]
+     *  [-] -  (Minus)   - [1]
+     *  [*] - (Multiply) - [2]
+     *  [/] -  (Divide)  - [3]
+     *
+     */
+
+    ShuntingClass() { }
+    ShuntingClass(String eq) {
+        this.eq = eq;
+    }
+
+    public void setEq(String eq) {
+        this.eq = eq;
+    }
+
+    public String getEq() {
+        return eq;
+    }
+
+    public void setVars(Map<String, String> vars) {
+        this.vars = vars;
+    }
+
+    public String replaceVars(String token){
+        //ArrayList<String> cleanTokenList = new ArrayList<>();
+        String currentVar = vars.get(token);
+        if(Pattern.matches("[a-z]", currentVar)) {
+           currentVar = replaceVars(currentVar);
+        }
+            return currentVar;
+    }
+
+    public String findSolution( String postfixEq ) {
+        Deque<String> stack = new ArrayDeque<String>();
+        for(String token: postfixEq.split("[ ]")){
+            if(Pattern.matches("[0-9]+", token)){
+                stack.push(token);
+            }else {
+                int a = Integer.parseInt(stack.pop());
+                int b = Integer.parseInt(stack.pop());
+                int res = 0;
+                switch (token){
+                    case "+":
+                        res = b + a;
+                        break;
+                    case "-":
+                        res = b - a;
+                        break;
+                    case "*":
+                        res = b * a;
+                        break;
+                    case "/":
+                        res = b / a;
+                        break;
+                }
+                stack.push(String.valueOf(res));
+            }
+        }
+        return stack.pop();
+    }
+
+    String[] parseEq(String infixEq){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/app/thejerome/IFMO_Servlets_20190915/pullRequest?prid=4502470)